docs/application/native/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1154 @@
+
+
+The following figure illustrates the voice control manager life-cycle.
+
+**Figure: Voice control manager (left) and Voice control service (right) life-cycle states**
+
+![Voice control manager and Voice control service life-cycle states](./media/voice_control_manager.png)
+
+The following are the voice control service state:
+1.  The user starts recording for recognition by using a voice control manager application, button, or voice trigger. If the start is successful, the voice control service state changes to `VC_SERVICE_STATE_RECORDING`.
+2.  After the recording is completed, the service state changes to `VC_SERVICE_STATE_PROCESSING` for recognition processing.
+3.  After the recognition is completed, the service state changes to `VC_SERVICE_STATE_READY`.
+
+## Prerequisites
+
+To enable your application to use the voice control manager functionality:
+
+1.  To use the Voice control manager API, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+    ```xml
+    <privileges>
+        <privilege>http://tizen.org/privilege/recorder</privilege>
+        <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+    </privileges>
+    ```
+
+    > [!NOTE]
+    > To use this privilege, your application must be signed with a platform-level certificate.
+
+2.  To use the functions and data types of the Voice control manager API (in [mobile](../../api/mobile/latest/group__CAPI__UIX__VOICE__CONTROL__MANAGER__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__UIX__VOICE__CONTROL__MANAGER__MODULE.html) applications), include the `<voice_control_manager.h>` header file in your application:
+
+    ```c
+    #include <voice_control_manager.h>
+    ```
+
+3.  Initialize the voice control with `vc_mgr_initialize()`:
+
+    ```c
+    void
+    initialize_voice_control_manager()
+    {
+        int ret;
+        ret = vc_mgr_initialize();
+        if (VC_ERROR_NONE != ret)
+            /* Error handling */
+    }
+    ```
+
+    If the function call is successful, the voice control state changes to `VC_STATE_INITIALIZED`.
+
+    > [!NOTE]
+    > The voice control feature is not thread-safe and depends on the Ecore main loop. Implement voice control within the Ecore main loop and do not use it in a thread.
+
+3.  Prepare the voice control service with `vc_mgr_prepare()`, which connects the background voice control daemon. The daemon records and recognizes audio data and converts sound to text:
+
+    ```c
+    void
+    prepare_voice_control_manager()
+    {
+        int ret;
+        ret = vc_mgr_prepare();
+        if (VC_ERROR_NONE != ret)
+            /* Error handling */
+    }
+    ```
+
+    `vc_mgr_prepare()` is asynchronous. When the preparation succeeds, the voice control manager state changes from `VC_STATE_INITIALIZED` to `VC_STATE_READY`. The error callback is triggered if `vc_mgr_prepare()` fails.
```

Comment:
>Can you explain which function name has 'the'?
The first vc_mgr_prepare() starts with 'the', but the next vc_mgr_prepare() has no 'the'.
I do not know why.

the is used when there is a classifier. 

Like 

The  `vc_mgr_prepare` property is defined...
As per our style guide, if there is a function name like prepare(), we do not write it like 
The prepare() function, instead we write prepare() . As a developer explicitly reads it as prepare function, when he/she sees ()

So when the function(classifier) is removed, the becomes irrelevant.